docs/Encyclopedia/pages/BCGSC_miRNA_Profiling.md

```diff
@@ -0,0 +1,31 @@
+# BCGSC miRNA Profiling
+
+## Description ##
+
+BCGSC miRNA Profiling is a miRNA profiling pipeline used in GDC for miRNA quantification analysis.
+
+## Overview ##
+
+BCGSC miRNA Profiling is a part of the miRNA Analysis Pipeline in the GDC miRNA quantification analysis. It uses tumor or normal alignments and generates Isoform Expression Quantification and miRNA Expression Quantification data.
+
+### Input
+
+* GDC-aligned BAM - miRNA-Seq
+
+### Output
+
+* TSV (Data Type: Isoform Expression Quantification, miRNA Expression Quantification)
```

Comment:
Recommendation: Change this line to "TSV/TXT" and remove the one below.  Hopefully we can get this standardized soon.